modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc

```diff
@@ -14,7 +14,8 @@ The following environment variable is mandatory:
 - `BONITA_API_URL`: The URL for the Bonita API.
 
 The following environment variables are optional:
-
+- `BONITA_HEALTHCHECK_USER`: Defines a custom username (default: `monitoring`).
+- `BONITA_HEALTHCHECK_PASSWORD`: Defines a custom password (default: `mon1tor1ng_adm1n`).
```

Comment:
It is a bit obvious with the names of those variables but this description is more accurate I guess:
```suggestion
- `BONITA_HEALTHCHECK_USER`: Defines the username to request the Bonita healthcheck API (default: `monitoring`).
- `BONITA_HEALTHCHECK_PASSWORD`: Defines the password to request the Bonita healthcheck API (default: `mon1tor1ng_adm1n`).
```